src/System.Net.Mail/src/System/Net/Mail/SmtpClient.cs

```diff
@@ -311,6 +314,25 @@ public int Timeout
             }
         }
 
+        public ServicePoint ServicePoint
+        {
+            get
+            {
+                CheckHostAndPort();
+                if (_servicePoint == null)
+                {
+                    // This differs from desktop, where it uses an internal overload of FindServicePoint that just
```

Comment:
Instead of explaining here (which can easily get out of date), we could add a corefx and net463 test showing the difference in behavior.